modules/account-lib/test/unit/coin/avaxc/transactionBuilder/walletInitializationBuilder.ts

```diff
@@ -147,24 +148,13 @@ describe('AvaxC Wallet Initialization Builder', function () {
       should.deepEqual(tx.id, txFromRaw.id);
     });
 
-    it('Should getting same tx from raw tx ETH', async function () {
-      txBuilder = getBuilder('eth') as AvaxC.TransactionBuilder;
-      txBuilder.fee({
-        fee: '280000000000',
-        gasLimit: '7000000',
-      });
-      txBuilder.counter(1);
-      txBuilder.type(TransactionType.WalletInitialization);
+    it('unsigned transaction with final v check', async function () {
```

Comment:
[np] `it` tests should be followed by should. otherwise use `test`

`import { test } from 'mocha'`